ast/desugar/Desugar.cc

```diff
@@ -704,29 +704,30 @@ class DuplicateHashKeyCheck {
     }
 };
 
-ExpressionPtr node2TreeImpl(DesugarContext dctx, unique_ptr<parser::Node> what) {
+// Translate a tree to an expression. NOTE: this should only be called from `node2TreeImpl`.
+ExpressionPtr node2TreeImplBody(DesugarContext dctx, parser::Node *what) {
```

Comment:
Not attached to this name, I just needed something that was different from `node2TreeImpl`.